### DISCUSSION TITLE
Which UI framework whould I choose for migrating my WPF application to linux based application?

### DISCUSSION THREAD

**Discussion by omerkibar** [unoplatform/uno#13363] [General]
Which UI framework whould I choose for migrating my WPF application to linux based application? The options are: Avalonia vs UNO

**Comment by sasakrsmanovic** [comment]
Hello - You've come to Uno Platform repo so our answer is pretty clear - Uno Platform. 

Open-ended, opinion seeking questions, especially when requirements are ambiguous, may be better suited for social media platform rather than GitHub repositories for any particular platform.
